### DISCUSSION TITLE
Feb 1

### DISCUSSION THREAD

**Discussion by ngsy** [GymHubCommunity/GymHub-FE#88] [DailyMeeting]
### ⚽️ 회의 내용

- [x] 개발 내용 공유 
  수: 토글 버튼 , 날짜 입력폼 개발 
  수영: 지도 출력 , 현재 위치 기능 구현 
  슬희:코드 리뷰 , 백엔드 분과 내일 부터 이야기 해볼 예정 
  윤혁:토글 메뉴 버튼 구현 

### 🏀 오늘 할 일

수: 리엑트 훅 폼 에러 수정 , 날짜 입력폼 수정 
수영:남는거 골라서하겠습니다. 
슬희:항소 준비
윤혁:슬라이드 도전 해보겠습니다. 라이브러리 swiper 사용 

### 🎱 TMI

수: 무한스크롤 넘길까 고민, 날짜입력폼도 이뿐걸로 바꾸고싶다.. 3대 500 별거 아니다 선언(으샤으샤 운동 기준)
수영:기상시간을 6시로 바꿔야겠다.
슬희:4명이서 제주도 헐.. 2명이 도중에 복귀하셨다.. 여자 셋이 왔다고 했다가 슬희님이 보내버렸다.. 아이고.. 
윤혁: 윤혁님도 이성과 여행을갔다가 엄크가 뜬적이있다. 오 친구와 헬스장 1RM 측정 가즈아! 3대 500 예상

**Comment by hanseulhee** [comment]
제가 보낸 거 아닌데 !!!!
